package transMsg;

enum	EReturnStatus
{
	ERS_SUCCESS			= 0;
	
	ERS_SERVER_ERROR	= 1;
	ERS_LIB_ALREADDONE	= 2;
	
	ERS_ALREAD_LOGIN	= 3;
	
	ERS_ORDER_DENY			= 4;
	ERS_ORDER_OPERATE_FAIL	= 5;
	ERS_ORDER_NONACTIVE		= 6;
	
	ERS_USER_NONACTIVE	= 10;
	
	ERS_EMPTY_FILE		= 100;
	ERS_EMPTY_RECORD	= 101;
	ERS_EMPTY_USERINFO	= 102;
	ERS_EMPTY_STATE		= 103;
	
	ERS_WRONG_MESSAGE	= 200;
	ERS_WRONG_ACCESS	= 201;
	ERS_WRONG_PWD		= 202;
	ERS_WRONG_RECORD	= 203;
	ERS_WRONG_VERCODE	= 204;
	ERS_WRONG_FILE		= 205;
	ERS_WRONG_STATUS	= 206;
	
	ERS_CHARG_DENY		= 501;
}